examples/dbmgr.py

```diff
@@ -162,3 +169,36 @@ def removeDB(self):
         widget.deleteLater()
         # send the database list and a logging message
         self.sendDB(False, db.name)
+
+    @pyqtSlot()
+    def openCloseDatacalc(self):
+        """Opens or closes a datacalc app.
+
+        If the dataclac app is open, close it.
+        Otherwise, open a new datacalc app.
+        """
+        if self.openCloseDatacalcResult:
```

Comment:
Please make it explicit: `... is not None`, even if we have not defined the boolean value of the result objects.

Reply:
It looks more clear!